code/modules/mining/ores_coins.dm

```diff
@@ -317,26 +317,29 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 /obj/item/coin
 	icon = 'icons/obj/economy.dmi'
 	name = "coin"
-	icon_state = "coin"
+	icon_state = "coin" //If this coin has a unique sprite, this is changed on initialization.
 	flags_1 = CONDUCT_1
 	force = 1
 	throwforce = 2
 	w_class = WEIGHT_CLASS_TINY
 	custom_materials = list(/datum/material/iron = 400)
-	material_flags = MATERIAL_ADD_PREFIX | MATERIAL_COLOR
+	material_flags = MATERIAL_ADD_PREFIX
 	var/string_attached
 	var/list/sideslist = list("heads","tails")
 	var/cooldown = 0
 	var/value
 	var/coinflip
+	var/coin_icon = "greyscale" //This is used to define which coin type to use. If this isn't supplied, it uses material color.

```

Comment:
This needs to be part of the material datum if you want a unique sprite for coins. Material subtypes for coins aren't used for printed coins